### PR TITLE
EVG-6087 correctly display metric value labels, fix BF query

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: public/static/app/perf/TrendSamples.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ mciModule.factory('TrendSamples', function() {
           this.seriesByName[rec.name] = [];
         }
 
-        var maxThreadOpsPerSecItem = _.max(rec.results, function(d) {
-          return d.ops_per_sec
+        var maxValues = _.max(rec.results, function(d) {
+          return null;
```
Comment (bsamek) [inline public/static/app/perf/TrendSamples.js]: What's the purpose of this change?

Comment (john-m-liu) [reply public/static/app/perf/TrendSamples.js]: It's to make it explicit that the return value doesn't matter
